cmd/lifecycle/cli/flags.go

```diff
@@ -168,6 +168,11 @@ func FlagForceRebase(force *bool) {
 	flagSet.BoolVar(force, "force", *force, "execute rebase even if operation is unsafe")
 }
 
+// FlagInsecureRegistry sets insecure-registry parameter as available
+func FlagInsecureRegistry(insecureRegistry *str.Slice) {
```

Comment:
I think according to the RFC we also want to add this to: exporter, restorer, rebaser

Reply:
You are right, I wanted first to be sure of doing the right thing in the creator.
I pushed some changes and now I'm gonna work on the other parts starting from some tests, thanks!